src/adapters/futures_io.rs

```diff
@@ -64,6 +64,21 @@ impl<T: futures::io::AsyncWrite + Unpin + ?Sized> crate::asynch::Write for FromF
     }
 }
 
+impl<T: futures::io::AsyncSeek + Unpin + ?Sized> crate::asynch::Seek for FromFutures<T> {
+    type SeekFuture<'a> = impl Future<Output = Result<u64, Self::Error>>
+    where
+        Self: 'a;
+
+    fn seek<'a>(&'a mut self, pos: crate::SeekFrom) -> Self::SeekFuture<'a> {
+        let pos = match pos {
```

Comment:
`futures::io::SeekFrom` is a reexport of `std::io::SeekFrom` so you can use the conversion impls you did, with `.into()`

Reply:
oh you just did, nice!